packages/xstate-test/src/index.ts

```diff
@@ -1,422 +1,23 @@
-import {
-  getPathFromEvents,
-  getShortestPaths,
-  getSimplePaths,
-  getStateNodes,
-  StatePathsMap,
-  ValueAdjMapOptions
-} from '@xstate/graph';
-import { StateMachine, EventObject, State, StateValue } from 'xstate';
-import slimChalk from './slimChalk';
-import {
-  TestModelCoverage,
-  TestModelOptions,
-  TestPlan,
-  StatePredicate,
-  TestPathResult,
-  TestSegmentResult,
-  TestMeta,
-  EventExecutor,
-  CoverageOptions
-} from './types';
+import { EventObject } from 'xstate';
```

Comment:
I believe this file is missing the export of `TestModel`.

Reply:
Fixed here: [57acfad](https://github.com/statelyai/xstate/pull/3036/commits/57acfadd5ec27ca329eb6061f94353110460e003)